app/services/fuzzySearchService.js

```diff
@@ -79,6 +81,11 @@ const filterByAmounts = (results, amounts) => {
   })
 }
 
+const filterByCounty = (searchResults, counties) => {
+  if (!counties || !counties.length) return searchResults
+  return searchResults.filter(x => counties.includes(x.county_council))
```

Comment:
Just noticed this but we need to make sure that the comparisons are case insensitive. Please look at the filterByScheme function for example.

Reply:
Also, it would be better to call the function filterByCounties because it filters results based on more than one county. Hope that makes sense.

Reply:
ok, updated the filter to match case insensitive comparison

Reply:
White integrating these changes with 46, i have just noticed that it didn't return any results when filtering via counties. And it turns out the test data I was using locally had a spare in front of the county_council, for example ' Herfordshire' or ' West Sussex'. Do you know if test data could have this issue? In that case, we might need to trim white spaces when filtering by schemes and counties 🤔

Reply:
I can put trim() here to solve the current issue, but this may fail in other places also if the actual data issue is not solved/sanitised,  like search etc.. So we need to deal with actual problem. As discussed let us raise another ticket to deal with the actual data issue separately.